database/schema.sql

```diff
@@ -207,6 +207,72 @@ CREATE TABLE rotation_move_history (
   FOREIGN KEY (new_bucket) REFERENCES rotation_buckets (bucket_id)
 );
 
+-- BUDGET TABLES
+
+-- Table of all accounts Ruddock owns.
+CREATE TABLE budget_accounts (
+  account_id INTEGER NOT NULL AUTO_INCREMENT,
+  account_name VARCHAR(32) NOT NULL,
```

Comment:
VARCHAR(255)? No good reason to artificially limit the length, since anything <=255 will have the same size on disk.